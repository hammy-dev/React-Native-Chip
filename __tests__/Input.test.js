import React from 'react';
import renderer from 'react-test-renderer';
import SimpleInput from '../src/components/Input/Input';

describe('Input', () => {
    it('renders correctly', () => {
        const tree = renderer.create( <SimpleInput title={"test"} />).toJSON();
      expect(tree.type).toBe( 'View')
    });
   
  });
  