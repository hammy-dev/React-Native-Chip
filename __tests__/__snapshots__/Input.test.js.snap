// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  placeholder="First Name"
  placeholderTextColor="#9e9e9e"
  rejectResponderTermination={true}
  secureTextEntry={false}
  style={
    Object {
      "borderBottomColor": undefined,
      "borderBottomWidth": 1,
      "marginVertical": 11,
      "width": "100%",
    }
  }
  underlineColorAndroid="transparent"
/>
`;
